src/Views/scstormrunoff.html

```diff
@@ -134,10 +134,131 @@ <h4>Citation</h4>
                         </div>
                         <!-- Synthetic Urban Hydrograph tab -->
                         <div ng-if="vm.SelectedTab == 3" style="position:relative;">
-                            <div> Coming Soon. </div>
-                            <br>
+                            <div>&nbsp;</div>
+                            <form id="SyntheticUrbanHydrograph" name="SyntheticUrbanHydrograph" class="hidden-print" novalidate>
+                                <label for="AEPOptionsSynthetic">Select an Annual Exceedance Probability (AEP):</label>
+                                <select name="AEPOptionsSynthetic" ng-model="vm.SelectedAEPSynthetic" ng-options="precip.name for precip in vm.AEPOptionsSynthetic track by precip.value"></select>
+                                <label for="StandardCurve">Select a Standard Curve Number Value</label>
+                                <select name="StandardCurve" ng-model="vm.SelectedStandardCurve" ng-options="curve.name for curve in vm.StandardCurveOptions track by curve.value"></select>
+                                <label for="CNModification">Select a CN Modification Method</label>
+                                <select name="CNModification" ng-model="vm.SelectedCNModification" ng-options="cnmod.name for cnmod in vm.CNModificationOptions track by cnmod.value"></select>
+                                <label for="RainfallDistribution">Select a Rainfall Distribution Curve</label>
+                                <select name="RainfallDistribution" ng-model="vm.SelectedRainfallDistribution" ng-options="distr.name for distr in vm.RainfallDistributionOptions track by distr.value"></select>
+                                <label for="DrainageAreaSynthetic">Drainage Area (square miles):</label> <span ng-if="SyntheticUrbanHydrograph.DrainageAreaSynthetic.$error.pattern" class="red label"><span class="fa fa-exclamation-triangle"></span>&nbsp;Value must be greater than 0</span>
+                                <input type="number" name="DrainageAreaSynthetic" ng-model="vm.drainageAreaSynthetic" ng-minlength="1" ng-pattern="greaterThanZero" required/>
+                                
+                                <span class="form-row">
```

Comment:
This section (Peak Rate Factor, Standard Curve Number, Watershed Retention, Initial Abstraction) is slightly indented. Can we get it in line with the rest of the form elements?